script/Arcueid/Day 6/06_00_ARC06_2_2.txt

```diff
@@ -348,6 +348,7 @@ It's on par with the care I received from the old man from the clinic, who took
 [sha:acde0b55ba9a0e7317e101d55d915d134e432b99]{
 -- Page 239, Offset 14491.
 -- 博士と呼ばれた事がよほど嬉しいらしい。
+// 阿良句先生 - Arach-sensei, 阿良句医師 - Dr. Arach, 阿良句博士 - Prof. Arach, 阿良句女史 - Ms. Arach,阿良句氏 - Arach-shi, go through everything including day 2
```

Comment:
Not it

Reply:
For now let's just get this file fixed and when we re-TLC days 1/2 (the ones most desperately in need of love), we'll just keep this in mind. I know I won't be forgetting this trauma any time soon